.gitignore

```diff
@@ -8,9 +8,6 @@
 # testing
 /coverage
 
-# production
-/build
```

Comment:
Why delete this?

Reply:
Resolved, deleted to push build, now it works! added it back